airbyte-scheduler/app/src/main/java/io/airbyte/scheduler/app/SchedulerApp.java

```diff
@@ -113,6 +114,8 @@ public SchedulerApp(final Path workspaceRoot,
   }
 
   public void start() throws IOException {
+    final Configs configs = new EnvConfigs();
+    final WorkerConfigs workerConfigs = new WorkerConfigs(configs);
```

Comment:
I believe this should be safe to do here in the SchedulerApp, because I see that the resource requirement env variables exist in each of these environments, and with the change I made in `EnvConfigs` above they should always be populated with default values if they are no overridden by the user:
- Docker compose: https://github.com/airbytehq/airbyte/blob/84d1456fcf6103fecd091b1a4cb24b0195f660d1/docker-compose.yaml#L67-L70
- Kube overlays: https://github.com/airbytehq/airbyte/blob/84d1456fcf6103fecd091b1a4cb24b0195f660d1/kube/overlays/stable/.env#L57-L60
- Helm charts: https://github.com/airbytehq/airbyte/blob/84d1456fcf6103fecd091b1a4cb24b0195f660d1/charts/airbyte/templates/scheduler/deployment.yaml#L127-L146